ai2_kit/algorithm/proton_transfer.py

```diff
@@ -157,38 +157,75 @@ def analysis(self, initial_donor: int, out_dir: str):
 
 
 def proton_transfer_detection(
-        input_traj: str,
-        out_dir: str,
-        cell: List[float],
-        acceptor_elements: List[str],
-        initial_donors: List[int],
-        core_num: int = 4,
-        dt: float = 0.0005,
-        r_a: float = 4.0,
-        r_h: float = 1.3,
-        rho_0: float = 1 / 2.2,
-        rho_max: float = 0.5,
-        max_depth: int = 4,
-        g_threshold: float = 0.0001,
+    input_traj: str,
+    out_dir: str,
+    cell: List[float],
+    acceptor_elements: List[str],
+    initial_donors: List[int],
+    core_num: int = 4,
+    dt: float = 0.0005,
+    r_a: float = 4.0,
+    r_h: float = 1.3,
+    rho_0: float = 1 / 2.2,
+    rho_max: float = 0.5,
+    max_depth: int = 4,
+    g_threshold: float = 0.0001,
 ):
+    """
+    cli entry for proton transfer detection
+    """
+    # setup universe from trajectory
+    universe = Universe(input_traj)
+    universe.trajectory.ts.dt = dt
+    universe.dimensions = np.array(cell)
+
+    proton_transfer_detection_lower(
```

Comment:
The `proton_transfer_detection_lower` is supposed to be `proton_transfer_detection`